android-components/components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -255,6 +262,47 @@ class BrowserIcons constructor(
         }
     }
 
+    @Composable
+    fun LoadIntoViewInternal(
```

Comment:
Let's rename this to ditch the `View` and make it more clear that this is Composable UI and not just a business logic function with the `@Composable` tag.

```suggestion
    fun LoadableImage(
```